package.json

```diff
@@ -43,7 +43,8 @@
       "<rootDir>/node_modules/"
     ],
     "testMatch": [
-      "**/test/?(*.)+(spec|test).[jt]s?(x)"
+      "**/test/?(*.)+(spec|test).[jt]s?(x)",
+      "**/dev-test/?(*.)+(spec|test).[jt]s?(x)"
```

Comment:
We need to disable this line in order for tests on CI to work properly.

Reply:
Change ;)